pkg/portal/model/app.go

```diff
@@ -27,6 +27,6 @@ func (a *App) LoadFile(path string) ([]byte, error) {
 }
 
 type AppConfigFile struct {
-	Path    string
+	Name    string
```

Comment:
If we accept file name, we cannot express `/templates/zh/forgot_password_email.html`.

Maybe we need to adopt [an encoding scheme](https://github.com/facebook/flow/blob/v0.133.0/src/hack_forked/utils/sys/path.ml#L66) to encode the key so that the encoded is valid k8s configmap/secret key.

Reply:
It'll be hard to migrate later, so I'll add encoding here now.